<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestParamDialog</class>
 <widget class="QDialog" name="TestParamDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>425</width>
    <height>326</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="paramGroupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>231</width>
     <height>221</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>211</width>
      <height>211</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="startAddrLabel">
         <property name="text">
          <string>读写地址：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="startAddrLineEdit"/>
       </item>
       <item>
        <widget class="QLabel" name="startMbLabel">
         <property name="text">
          <string>0MB</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QSlider" name="startAddrSlider">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="lengthLabel">
         <property name="text">
          <string>读写长度：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lengthLineEdit"/>
       </item>
       <item>
        <widget class="QLabel" name="lengthMbLabel">
         <property name="text">
          <string>0MB</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QSlider" name="lengthSlider">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="countLabel">
         <property name="text">
          <string>读写次数：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="countLineEdit"/>
       </item>
       <item>
        <widget class="QLabel" name="count2Label">
         <property name="text">
          <string>次 </string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QCheckBox" name="verifyCheckBox">
         <property name="text">
          <string>数据校验</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="wholeChipCheckBox">
         <property name="text">
          <string>整片测试</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="chipRangeLabel">
       <property name="text">
        <string>整片范围：0x00-0x08000000</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="enterBtn">
         <property name="text">
          <string>确定</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
